src/welcomePage/WelcomePageOnboarding/hooks/useOnboardingData.tsx

```diff
@@ -22,10 +23,12 @@ const getStepsData = ({
   intl,
   isStepCompleted,
   onStepComplete,
+  posthogCapture,
```

Comment:
when you have abstraction, then you should not refer to the name "Posthog" anywhere apart the file where you build the abstraction